README.md

```diff
@@ -0,0 +1,84 @@
+Windows Agent Installer module for Jenkins
+---
+
+This module is a part of the Jenkins core, which provides features for agent management as Windows services.
+
+Provided features:
+
+* Installation agents as Windows services powered by the [Windows Service Wrapper (WinSW)](https://github.com/kohsuke/winsw). 
+* GUI option for the JNLP agents, which adds the `Install as a service` menu option.
+* Automatic upgrade of Windows service wrapper versions to keep them aligned with the version from this module.
+
+## Changelog
+
+See the changelog [here](./CHANGELOG.md).
+
+## Installation 
+
+### Automatic
+
+Once you start a JNLP agents on Windows in the GUI mode (e.g. via Java WebStart), 
+It will display the `Install as a service` menu option once Jenkins master finishes the slave configuration.
```

Comment:
Suggest "It will display the `Install as a service` menu option once the Jenkins master finishes the agent configuration."